mk/update-travis-yml.py

```diff
@@ -145,16 +150,12 @@ def prefix_all(prefix, xs):
         packages = sorted(get_linux_packages_to_install(target, compiler, arch, kcov))
         sources_with_dups = sum([get_sources_for_package(p) for p in packages],[])
         sources = sorted(list(set(sources_with_dups)))
-
-    # TODO: Use trusty for everything?
-    if arch in ["aarch64", "arm", "armv7"]:
         template += """
-      dist: trusty
-      sudo: required"""
+      dist: trusty"""
 
     if sys == "linux":
```

Comment:
This `if` can be combined with the previous `if`.

Reply:
I merged it with the previous and the next if.